.github/workflows/e2e.yml

```diff
@@ -0,0 +1,59 @@
+name: e2e
+on:
+  workflow_dispatch:
+    inputs:
+      firefox-version:
+        description: The version of selenium/standalone-firefox image to use
+        default: latest
+        required: true
+      chromium-version:
+        description: The version of selenium/standalone-chrome image to use
+        default: latest
+        required: true
+      kubo-version:
+        description: The version of ipfs/kubo image to use
+        default: latest
+        required: true
+      ipfs-companion-version:
+        description: The version of ipfs-companion extension to use (defaults to building the extension from source)
+        default: ''
+        required: false
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+      - name: Set up node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Download ipfs-companion
+        if: inputs.ipfs-companion-version != ''
+        run: ./ci/download-release-artifacts.sh
+        env:
+          IPFS_COMPANION_VERSION: ${{ inputs.ipfs-companion-version }}
+      - name: Build ipfs-companion
+        if: inputs.ipfs-companion-version == ''
+        run: npm run release-build
```

Comment:
we should eventually upload the built docker image (`ipfs-companion-release-build`) and reuse that if possible.. (hash of source or non-docker build artifacts to determine if we want to use the same image?)

Reply:
I created https://github.com/ipfs/ipfs-companion/issues/1132 for this